code/modules/instruments/songs/_song.dm

```diff
@@ -153,6 +153,7 @@
 	stop_playing()
 	SSinstruments.on_song_del(src)
 	lines = null
+	using_instrument.songs_using -= src
```

Comment:
Are you confident `using_instrument` always exists?

Reply:
Yes, I tried to break it in testing and failed